staking-miner-playground/runtime/src/lib.rs

```diff
@@ -448,17 +432,20 @@ impl pallet_staking::Config for Runtime {
 	type BenchmarkingConfig = StakingBenchmarkingConfig;
 	type HistoryDepth = HistoryDepth;
 	type TargetList = pallet_staking::UseValidatorsMap<Self>;
+	type NominationsQuota = pallet_staking::FixedNominationsQuota<{ MaxNominations::get() }>;
 	/// A super-majority of the council can cancel the slash.
 	type AdminOrigin = EnsureRoot<AccountId>;
+	type EventListeners = ();
 }
 
 parameter_types! {
 	pub const StakingUnsignedPriority: TransactionPriority = TransactionPriority::max_value() / 2;
 
 	// signed config
 	pub const SignedRewardBase: Balance = 1 * DOLLARS;
-	pub const SignedDepositBase: Balance = 1 * DOLLARS;
+	pub const SignedFixedDeposit: Balance = 1 * DOLLARS;
 	pub const SignedDepositByte: Balance = 1 * CENTS;
+	pub const SignedDepositIncreaseFactor: Percent = Percent::from_percent(10);
```

Comment:
I don't know about this offhand; how do you know which value to set?

Reply:
I copied it from the westend/polkadot runtime but @gpestana can you double check  :)